pkg/local-storage/member/node/qos/cgroups.go

```diff
@@ -25,10 +25,9 @@ func NewVolumeCgroupsManager() (VolumeCgroupsManager, error) {
 	mode := cgroups.Mode()
 	switch mode {
 	case cgroups.Legacy:
-		return &cgroupV1{nsexecutor.New()}, nil
+		return &cgroupV1{}, nil
 	case cgroups.Unified, cgroups.Hybrid:
-		// TODO: support cgroup v2
```

Comment:
For hybird mode, the cgroup2 fs is mounted on `/sys/fs/cgroup/unified`

Reply:
Yes! You're right. Should we use the v1 or v2 API when hybrid mode is enabled?

Reply:
v2 is preferred

Reply:
Alright. I fixed the path for hybrid mode